### ISSUE TITLE
Could not find a declaration file for module '@viselect/react'

### ISSUE THREAD

**Issue by rlesniak** [simonwep/selection#203]
#### What is the problem?
TS cant find typings

#### What is the current behavior?
```
Could not find a declaration file for module '@viselect/react'. 'blabla/node_modules/@viselect/react/dist/viselect.umd.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/viselect__react` if it exists or add a new declaration (.d.ts) file containing `declare module '@viselect/react';`
```

#### What is the expected behavior?
Working TS typings.

#### Your environment:
```
Toolset (e.g. webpack, vite, vue-cli...): webpack, nx
Version (@viselect/<insert package>): @viselect/react 3.3.0
Browser:  Chrome latest
OS:  macOS Ventura
```

package.json's `types` points to `"types": "./dist/index.d.ts",` which does not exist, it is inside `src/` folder so that should be solution

**Comment by rlesniak** [comment]
I see that `vanilla` version has the same issue

**Comment by simonwep** [comment]
Thanks, should be fixed in v3.3.1 - don't mind the failed pipeline, I'm working on it. I published the patch manually :)

**Comment by rlesniak** [comment]
Wow, that was quick. Thanks! :)
